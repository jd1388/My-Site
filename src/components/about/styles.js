import styled from 'styled-components';

import Colors from '../../styles/Colors';

export const Hero = styled.h1`
    font-size: 90pt;
    color: ${Colors.blue};
    font-weight: 400;
    padding-top: 15vh;
    margin-bottom: 0;
    align-self: center;

    @media (min-width: 800px) {
        font-size: 160pt;
        padding-top: 20vh;
    }
`;

export const AboutContainer = styled.div`
    display: flex;
    flex-direction: column;
    align-items: center;
    padding: 0 10%;
    height: 95vh;
    min-height: 800px;
    font-family: Roboto, sans-serif;

    @media (min-width: 800px) {
        padding: 0 25%;
        min-height: 1000px;
    }
`;

export const SectionContent = styled.div`
    text-align: justify;
    font-weight: 300;
    color: ${Colors.black};
    flex-grow: 1;
    display: flex;
    align-items: center;
`;
