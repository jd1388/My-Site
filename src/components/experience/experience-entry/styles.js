import styled from 'styled-components';

import Colors from '../../../styles/Colors';

export const ExperienceEntryContainer = styled.div`
    
    margin-bottom: 32px;
    display: grid;
    grid-template-columns: 1fr;


    &:last-child {
        margin-bottom: 0;
    }

    @media (min-width: 800px) {
        grid-template-columns: 30% 5% 65%;
    }
`;

export const ExperienceEntryHeader = styled.h4`
    font-weight: normal;
    color: ${Colors.white};
    text-align: left;
    font-size: 12pt;
    margin: 0;
    grid-column: 1;
    margin-bottom: 8px;

    @media (min-width: 800px) {
        font-size: 16pt;
    }
`;

export const ExperienceEntryContent = styled.p`
    font-weight: 300;
    text-align: justify;
    margin: 0;
    grid-column: 1;
    font-size: 12pt;

    @media (min-width: 800px) {
        grid-column: 3;
        font-size: 16pt;
    }
`;

export const ExperienceEntryDates = styled(ExperienceEntryContent)`
    grid-column: 1;
    text-align: left;
    margin-bottom: 8px;

    @media (min-width: 800px) {
        grid-row: 1;
        margin-bottom: 0;
    }
`;

export const ExperienceEntryPosition = styled(ExperienceEntryHeader)`
    grid-row: 2;

    @media (min-width: 800px) {
        grid-column: 1;
    }
`;
