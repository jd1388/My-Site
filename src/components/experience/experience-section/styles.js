import styled from 'styled-components';
import Colors from '../../../styles/Colors';

export const ExperienceSectionHeader = styled.h3`
    font-weight: 300;
    color: ${Colors.white};
    font-size: 18pt;

    @media (min-width: 800px) {
        font-size: 22pt;
    }
`;
