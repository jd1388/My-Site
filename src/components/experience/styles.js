import styled from 'styled-components';

import Colors from '../../styles/Colors';

export const ExperienceContainer = styled.div`
    align-items: center;
    background-color: ${Colors.blue};
    box-shadow: 0px 4px 20px 0px rgba(0,0,0,0.75);
    color: ${Colors.white};
    display: flex;
    font-family: Roboto, sans-serif;
    justify-content: center;
    min-height: 95vh;
    padding: 5vh 0;
    width: 100%;
`;

export const ExperienceContent = styled.div`
    padding: 50px 10%;

    @media (min-width: 800px) {
        padding: 50px 25%;
    }
`;

export const ExperienceHeader = styled.h2`
    color: ${Colors.white};
    font-size: 36pt;
    font-weight: normal;
    margin: 0;
    text-align: center;

    @media (min-width: 800px) {
        font-size: 48pt;
    }
`;
