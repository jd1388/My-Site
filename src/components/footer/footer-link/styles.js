import styled from 'styled-components';

import Colors from '../../../styles/Colors';

export const Link = styled.a`
    color: ${Colors.white};
    display: flex;
    font-weight: 300;
    text-decoration: none;
`;

export const LinkText = styled.span`
    padding-left: 5px;
    padding-top: 2px;
`;
