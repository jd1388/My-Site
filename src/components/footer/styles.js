import styled from 'styled-components';
import LinkedinIcon from 'mdi-react/LinkedinIcon';
import GithubCircleIcon from 'mdi-react/GithubCircleIcon';
import EmailIcon from 'mdi-react/EmailIcon';

import Colors from '../../styles/Colors';

export const FooterContainer = styled.div`
    align-items: center;
    background-color: ${Colors.gray};
    height: 5vh;
    min-height: 100px;
    color: ${Colors.white};
    display: flex;
    justify-content: space-around;
    padding: 0 10%;
    font-size: 12pt;

    @media (min-width: 800px) {
        padding: 0 25%;
    }
`;

export const StyledLinkedinIcon = styled(LinkedinIcon)`
    fill: ${Colors.white};
    height: 20px;
    width: 20px;
`;

export const StyledGithubIcon = styled(GithubCircleIcon)`
    fill: ${Colors.white};
    height: 20px;
    width: 20px;
`;

export const StyledEmailIcon = styled(EmailIcon)`
    fill: ${Colors.white};
    height: 20px;
    width: 20px;
`;
