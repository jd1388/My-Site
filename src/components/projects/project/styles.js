import styled from 'styled-components';

import Colors from '../../../styles/Colors';

export const ProjectContainer = styled.div`
    padding-bottom: 5px;

    &:last-child {
        padding-bottom: 0;
    }
`;

export const ProjectName = styled.h3`
    font-weight: 300;
    font-size: 18pt;
    color: ${Colors.blue};

    @media (min-width: 800px) {
        font-size: 22pt;
    }
`;

export const ProjectDescription = styled.p`
    font-weight: 300;
    text-align: justify;
    font-size: 12pt;

    @media (min-width: 800px) {
        font-size: 16pt;
    }
`;
