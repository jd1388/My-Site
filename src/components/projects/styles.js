import styled from 'styled-components';

import Colors from '../../styles/Colors';

export const ProjectsContainer = styled.div`
    min-height: 95vh;
    background-color: ${Colors.silver};
    font-family: Roboto, sans-serif;
    color: ${Colors.white};
    display: flex;
    justify-content: center;
    align-items: center;
    transform: skewY(2deg);
    box-shadow: 0px 4px 20px 0px rgba(0,0,0,0.75);
    border: 1 solid transparent;
    padding: 5vh 0;
    filter: url('data:image/svg+xml;utf8,<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><filter id="gaussian_blur"><feGaussianBlur in="SourceGraphic" stdDeviation="0" /></filter></defs></svg>#gaussian_blur');
`;

export const ProjectsContent = styled.div`
    padding: 50px 10%;
    transform: skewY(-2deg);

    @media (min-width: 800px) {
        padding: 50px 25%;
    }
`;

export const ProjectsHeader = styled.h2`
    font-weight: normal;
    font-size: 36pt;
    margin: 0;
    text-align: center;
    color: ${Colors.blue};

    @media (min-width: 800px) {
        font-size: 48pt;
    }
`;
