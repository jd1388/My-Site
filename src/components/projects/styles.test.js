import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { ProjectsContainer, ProjectsContent, ProjectsHeader } from './styles';

describe('Projects Styles', () => {
    test('renders the ProjectsContainer correctly', () => {
        const wrapper = shallow(<ProjectsContainer />);

        expect(toJson(wrapper)).toMatchSnapshot();
    });

    test('renders the ProjectsContent correctly', () => {
        const wrapper = shallow(<ProjectsContent />);

        expect(toJson(wrapper)).toMatchSnapshot();
    });

    test('renders the ProjectsHeader correctly', () => {
        const wrapper = shallow(<ProjectsHeader />);

        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
