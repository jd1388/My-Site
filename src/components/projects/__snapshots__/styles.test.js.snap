// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects Styles renders the ProjectsContainer correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
    align-items: center;
    background-color: ",
          "#575757",
          ";
    border: 1 solid transparent;
    box-shadow: 0px 4px 20px 0px rgba(0,0,0,0.75);
    color: ",
          "#FFFFFF",
          ";
    display: flex;
    font-family: Roboto, sans-serif;
    justify-content: center;
    min-height: 95vh;
    padding: 5vh 0;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "usesTheme": false,
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Projects Styles renders the ProjectsContent correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bwzfXH",
        "isStatic": false,
        "rules": Array [
          "
    padding: 50px 10%;

    @media (min-width: 800px) {
        padding: 50px 25%;
    }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bwzfXH",
      "target": "div",
      "toString": [Function],
      "usesTheme": false,
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Projects Styles renders the ProjectsHeader correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-htpNat",
        "isStatic": false,
        "rules": Array [
          "
    color: ",
          "#0984E3",
          ";
    font-size: 36pt;
    font-weight: normal;
    margin: 0;
    text-align: center;

    @media (min-width: 800px) {
        font-size: 48pt;
    }
",
        ],
      },
      "displayName": "styled.h2",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-htpNat",
      "target": "h2",
      "toString": [Function],
      "usesTheme": false,
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
